src/amber/cli/commands/routes.cr

```diff
@@ -43,7 +43,16 @@ module Amber::CLI
       end
 
       private def parse_routes
-        File.read_lines("config/routes.cr").each do |line|
+        # parse each routes file in modules
+        Dir.glob(["src/modules/**/*_routes.cr"]) do |file|
```

Comment:
You can use `Dir.glob("src/modules/**/*_routes.cr")` as well

Reply:
Can we make `"src/modules/**/*_routes.cr"` a setting in the `amber.yml` file? This will allow some flexibility and avoid   hard coding this value

Reply:
Also what do you think of creating a method and removing   the comment. 

```ruby 
def parse_modules_routes_file
  Dir.glob(["src/modules/**/*_routes.cr"]) do |file|
   parse_routes_file(file)
  end
end
```